photography/demo/mail_templates.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="email_template_pro_event_quote" model="mail.template">
+        <field name="report_template_ids" eval="[(5, 0, 0)]" />
+    </record>
+
+    <record id="email_template_studio_quote" model="mail.template">
+        <field name="report_template_ids" eval="[(5, 0, 0)]" />
+    </record>
```

Comment:
hello
it was breaking the runbot test, when the runbot isntalled the demo data it would confirm a SO and send the PDF along with it and runbot would crash
so i did it as a separate commit d5c4159127c08b38d0d69a59274b993395f2f7d2
if you manage to get it green without this change, by all means go ahead!

runbot fail examlpe: https://runbot.odoo.com/runbot/build/53156837